sass/elements/_kbd.scss

```
@@ -0,0 +1,12 @@
+kbd {
+  font-size: 0.8rem;
+  padding-left: 0.1rem;
+  padding-right: 0.1rem;
+  padding-top: 0.1rem;
+  padding-bottom: 0.1rem;
```

```suggestion
  padding: 0.1rem;
```

I believe these can be merged.